driver/examples/src/test/resources/projects/cats/ideprobe.conf

```diff
@@ -0,0 +1,11 @@
+include required(classpath("ideprobe.conf"))
+
+probe {
+  workspace {
+    path = "https://github.com/typelevel/cats"
+  }
```

Comment:
this could be `workspace.path = ???`

Reply:
we accept empty workspace as default so this doesn't even have to be here.

Reply:
what doesn't need to be there? can somehow the path = ??? suffice?